metpy/plots/util.py

```diff
@@ -8,6 +8,28 @@
 from ..units import concatenate
 
 
+def resample_nn_1d(a, centers):
+    """Helper function that returns one-dimensional nearest-neighbor
+    indexes based on user-specified centers.
+    ----------
+    a : array-like
+        1-dimensional array of numeric values from which to
+        extract indexes of nearest-neighbors
+    centers : array-like
+        1-dimensional array of numeric values representing a subset of values to approximate
+    Returns
+    -------
+        An array of indexes representing values closest to given array values
+
+    """
+    ix = []
+    for center in centers:
+        index = (np.abs(a-center)).argmin()
```

Comment:
Can you put spaces around `-`? I'm not sure how it passed the pep8 check without them....